### ISSUE TITLE
Cannot get fresh install working

### ISSUE THREAD

**Issue by reckbo** [syl20bnr/spacemacs#6975]
After running into some errors last night I decided to make a fresh install, but simply cannot get it to install successfully -- I keep getting a 'package does not untar' and 'symbol's value as variable is void' errors, and they occur at different stages of the install when I restart (and restart, and restart ...)

# Steps

``` 
rm -rf .emacs.d
rm .spacemacs
git clone https://github.com/syl20bnr/spacemacs.git .emacs.d
# start emacs
``` 

# Errors

Here's one example:
````
Error (use-package): evil-search-highlight-persist :init: Symbol's function definition is void: global-evil-search-highlight-persist
Error (use-package): evil-surround :init: Symbol's function definition is void: global-evil-surround-mode
Warning (initialization): An error occurred while loading `/Users/ryan/.emacs.d/init.el':

Symbol's value as variable is void: ¦Û^ÏÛ¦

To ensure normal operation, you should investigate and remove the
cause of the error in your initialization file.  Start Emacs with
the `--debug-init' option to view a complete error backtrace.
```

# System

* Mac OS X El Capitan
* Emacs 24.5.1

**Comment by NJBS** [comment]
Can you try changing the value of `dotspacemacs-elpa-https` to `nil` in your `.spacemacs` file, wiping `.emacs.d/elpa`, then restarting emacs?

**Comment by sickill** [comment]
I'm having similar problem. First, I tried to upgrade existing installation. After restarting Emacs I started getting `Symbol's value as variable is void: \213`. Then I removed whole `.emacs.d/elpa` dir, restarted and then was getting some other errors when installing packages. So, finally I removed `.spacemacs` file and `.emacs.d` dir, made a fresh clone, and now it hangs on the following:

```
(Bootstrap) Installing evil...
(Bootstrap) Installing bind-map...
(Bootstrap) Installing bind-key...
(Bootstrap) Installing which-key...
(Bootstrap) Installing use-package...
```

with `Done (Total of 2 files compiled, 1 skipped)` in the status bar...
